### ISSUE TITLE
Feature request: skip version in PNG header

### ISSUE THREAD

**Issue by Aikhjarto** [matplotlib/matplotlib#13767]
# TL;DR
Is it possible to have an option to matplotlib's `savefig` to not include its version in the header of PNG files?

# Detailed problem description
I use matplotlib to generate plots and store them as PNGs. The PNGs are subsequently added in a git repository (for automatic generation of documentation and presentation slides).
Unfortunately, matplotlib stores its version string in the header of the PNG files. Therefore, with every version change of matplotlib (due update or different users using different versions), after re-running the code to generate the PNGs, git detects a file change, even for PNGs that are pixel-wise (and byte-wise) identical. This leads to either lot of manual work to exclude PNGs in which only the version string have changed, or rapidly growing git repos.
Therefore, it would come really handy to have an option to exclude matplotlib's version from the header of the generated PNGs.

**Comment by jklymak** [comment]
Sounds reasonable to me, but I guess the question is if the version was there due to some perceived need.  If so, maybe this suppression needs to be optional?

**Comment by anntzer** [comment]
You could just set it to whatever you want (`savefig(..., metadata={"Software": ""})`).

**Comment by Aikhjarto** [comment]
@anntzer Thanks for the hint, the `metadata` argument is suitable for my usecase.
